parquet-hadoop/src/main/java/org/apache/parquet/hadoop/metadata/ColumnChunkMetaData.java

```diff
@@ -141,11 +159,28 @@ public static ColumnChunkMetaData get(
           totalUncompressedSize);
     }
   }
+  
+  public static ColumnChunkMetaData getWithEncryptedMetadata(ParquetMetadataConverter parquetMetadataConverter, ColumnPath path, 
```

Comment:
I would expect some comments here why we need this object (to decrypt this metadata lazily instead of simply decrypt it at reading).